client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
+		selectedSite: getSelectedSite( state ),
```

Comment:
Just if there is a selected site, the only place left which checks for it is this https://github.com/Automattic/wp-calypso/pull/12944/files#diff-414303db60e403364a0ed6b10e351764R45.
We could also do this check on the selectedSiteId as well but it sound equivalent